dev/VSIX/ItemTemplates/Desktop/CSharp/BlankWindow/BlankWindow.xaml.cs

```diff
@@ -16,16 +16,15 @@
 // To learn more about WinUI, the WinUI project structure,
 // and more about our project templates, see: http://aka.ms/winui-project-info.
 
-namespace $rootnamespace$
+namespace $rootnamespace$;
+
+/// <summary>
+/// An empty window that can be used on its own or navigated to within a Frame.
+/// </summary>
+public sealed partial class $safeitemname$ : Window
 {
-    /// <summary>
-    /// An empty window that can be used on its own or navigated to within a Frame.
-    /// </summary>
-    public sealed partial class $safeitemname$ : Window
+    public $safeitemname$()
     {
-        public $safeitemname$()
-        {
-            this.InitializeComponent();
-        }
+        this.InitializeComponent();
```

Comment:
```suggestion
        InitializeComponent();
```
Forgot to call this out before, but general C# coding style guidance is to avoid using `this.`. See: https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md - Seems like something we should just ensure we follow here to set best practices for folks from the start.

Reply:
`this.` removed